### ISSUE TITLE
Fix unresolved links in 2.x docs

### ISSUE THREAD

**Issue by djspiewak** [typelevel/cats-effect#1839]
Run the build in `docs` and you'll see a bunch of unresolved links. They are, in fact, unresolved.

**Comment by antonkw** [comment]
@djspiewak 
Hello,
As far as I understood branch is 2.x.
What command I need to run to observe those errors.
I mean, you wrote `Run the build in ` but it's not clear what is that "build"
Thanks

**Comment by vasilmkd** [comment]
Hi @antonkw. Please run the following commands to build the documentation:
```bash
git checkout docs
git submodule update --init --recursive
./build.sh host
```

Here you would probably see some warnings regarding unresolved links. Hope this is helpful.

**Comment by antonkw** [comment]
@vasilmkd yes, I guess I reproduced that, thanks!

**Comment by antonkw** [comment]
The fix is just removing `.md` extensions in links.

@vasilmkd what is the branching model?

I need to fix files at `versions/2.x/docs`. As far as I understand I should raise PR against `series/2.x` branch. That's correct?

I didn't work with git submodules. What I should use to update `docs` (for testing purposes) with latest `series/2.x`?

Thanks

**Comment by vasilmkd** [comment]
Yes, open a PR against `series/2.x`.

If you want to test your solution, maybe you can try manually switching to the `docs` branch and updating the pages in `versions/2.x`? I think this is necessary because the submodules are created to link against `typelevel/cats-effect` which would not contain your changes. Not completely sure. Hope this helps.

**Comment by antonkw** [comment]
@vasilmkd 
Here it is: https://github.com/typelevel/cats-effect/pull/1965
Please let me know if anything is wrong.

**Comment by vasilmkd** [comment]
I saw it, I let the CI run :)

**Comment by vasilmkd** [comment]
Fixed in #2005.
